### DISCUSSION TITLE
[i18n] when do we start off with i18n?

### DISCUSSION THREAD

**Discussion by ryuujo1573** [Planshit/Tai#291] [Ideas]
Personally I'm happy to help on i18n, and I have some experiences on c-sharp also with mastery of some spoken languages. 

Maybe we can start from 🇬🇧 ?

**Comment by noberumotto** [comment]
非常感谢和期待你的帮助，这周末也就是周六周日我会开始准备

**Reply by noberumotto** [reply]
Thank you very much and look forward to your help. I will start preparations this weekend, which is Saturday and Sunday.
